contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -61,9 +72,76 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     return __numberOfWinners;
   }
 
+  function setPrizeSplit(MultipleWinnersPrizeSplit[2] memory prizeStrategySplit) external onlyOwner {
+    for (uint256 index = 0; index < prizeStrategySplit.length; index++) {
+        MultipleWinnersPrizeSplit memory split = prizeStrategySplit[index];
+
+        // If MultipleWinnersPrizeSplit is non-zero address set the prize split configuation.
+        if (split.target != address(0)) {
+            // Split percentage must be below 1000 and greater then 0 (e.x. 200 is equal to 20% percent)
+            // The range from 0 to 1000 is used for single decimal precision (e.x. 15 is 1.5%)
+            require(
+                split.percentage > 0 && split.percentage < 1000,
+                "MultipleWinners:invalid-prizesplit-percentage-amount"
+            );
+
+            // Emit PrizeSplitSet
+            PrizeSplitSet(split.target, split.percentage);
+
+            _prizeSplits[index] = split;
+        }
+    }
+  }
+
+  function prizeSplits() external view returns (MultipleWinnersPrizeSplit[2] memory) {
+    return _prizeSplits;
+  }
+
+  /**
+  * @dev Calculate the PrizeSplit percentage
+  * @param amount The prize amount
+  * @param percentage The prize split percentage amount
+  */
+  function _getPrizeSplitPercentage(uint256 amount, uint16 percentage) internal pure returns (uint256) {
```

Comment:
This function should be called `_getPrizeSplitAmount` no?
Since we assign it to `uint256 _splitAmount = _getPrizeSplitPercentage(_prizeTemp, split.percentage);` ?